python-main.js

```diff
@@ -481,6 +481,18 @@ function web_editor(config) {
     // the user sees their code or, in the case of a new program, uses some
     // sane defaults.
     function setupEditor(message, migration) {
+    
+        //to set the current language bold
```

Comment:
Missing a space after `//`
I also think this could be more descriptive to say something like `// Set the current language to bold in the dropdown menu`